showcase/src/app/welcome/pages/news/_news-page.component.scss

```diff
@@ -1,31 +1,93 @@
-.first-news-image {
-  background: url("/assets/images/screenshots/showcase-screenshot.png") no-repeat center center;
-  width: 250px;
-  height: 60px;
-  display: inline-block;
-  background-size: contain;
-}
-
-.second-news-image {
-  height: 60px;
-  width: 200px;
-
-  background-image: url(/assets/stark-core/logo/dark/stark_logo_dark.svg);
-  background-size: contain;
-  background-position: center;
-  background-repeat: no-repeat;
-}
-
-@media #{$desktop-query} {
-  .first-news-image {
-    width: 750px;
-    height: 200px;
+.fourth-news-image {
+	background: url("/assets/images/screenshots/10.0.0-beta3.PNG");
```

Comment:
Can you move all the images used for the news to `assets/images/news/` instead of `assets/images/screenshots` ? Otherwise it is confusing...